sdk/python/kfp/v2/compiler_tests/test_data/simple_pipeline_without_importer.json

```diff
@@ -0,0 +1,84 @@
+{
+  "pipelineInfo": {
+    "name": "simple-two-step-pipeline"
+  },
+  "tasks": [
+    {
+      "taskInfo": {
+	"name": "Write to GCS"
```

Comment:
The JSON indentation is a bit of malformed. Do you mind linting it?

Reply:
Done